tools/flang2/flang2exe/cgmain.cpp

```diff
@@ -13192,7 +13192,7 @@ process_formal_arguments(LL_ABI_Info *abi)
     /* Make a name for the real LLVM IR argument. This will also be used by
      * build_routine_and_parameter_entries(). */
     arg_op->string = (char *)ll_create_local_name(
-        llvm_info.curr_func, "%s%s", get_llvm_name(arg->sptr), suffix);
+       llvm_info.curr_func, "%s%s", SYMNAME(arg->sptr), suffix);
```

Comment:
The LLVM coding convention uses 4 spaces when indenting a wrapped parameter list like this.